molgenis-data-rest/src/test/java/org/molgenis/data/rest/RestControllerIT.java

```diff
@@ -112,7 +112,7 @@ public void getWithoutCountPermissionIsNotAllowed()
 		ValidatableResponse response = getWithToken("sys_sec_GroupAuthority", this.testUserToken);
 		response.statusCode(UNAUTHORIZED)
 				.body("errors.message[0]",
-						equalTo("No [COUNT] permission on entity type [Group authority] with id [sys_sec_GroupAuthority]"));
+						equalTo("No [READ] permission on entity type [Group authority] with id [sys_sec_GroupAuthority]"));
```

Comment:
Shouldn't this just stay COUNT? Since the function is named: getWithoutCountPermissionIsNotAllowed

Reply:
In order to be able to COUNT the 'Group authority' rows, you require READ access on the metadata.

Reply:
If you want to count Group Authority rows, you only need COUNT permission on the Group Authority EntityType. Let's add a test that tests that cause we may have actually broken that! However, this test case tests reading Group Authority rows using a GET request and therefore the message has actually improved.